src/Uno.Templates/content/unoapp/MyExtensionsApp.1/MyExtensionsApp.1.csproj

```diff
@@ -1,11 +1,14 @@
 <Project Sdk="Uno.Sdk">
   <PropertyGroup>
-    <!--#if (useWinAppSdk) -->
     <TargetFrameworks>$unoTargetFrameworks$;$(DotNetVersion)</TargetFrameworks>
+    <!--#if (useWinAppSdk) -->
     <TargetFrameworks Condition="$([MSBuild]::IsOSPlatform('windows')) or '$(EnableWindowsTargeting)' == 'true'">$(TargetFrameworks);$(DotNetVersion)-windows10.0.19041</TargetFrameworks>
-    <!--#else -->
-    <TargetFrameworks>$unoTargetFrameworks$;$(DotNetVersion)</TargetFrameworks>
     <!--#endif -->
+    <!--
+      Under Linux we can't restore ios or maccatalyst workloads are not available on linux https://github.com/dotnet/runtime/issues/85505
+      dotnet restore would fail if they are not removed from the list
+    -->
+    <TargetFrameworks Condition="$([MSBuild]::IsOSPlatform('linux')) == 'true'">$(TargetFrameworks.Replace("$(DotNetVersion)-ios","").Replace("$(DotNetVersion)-maccatalyst",""))</TargetFrameworks>
```

Comment:
Since the template generates the list of TFS (see $unoTargetFrameworks$) rather than doing a replace here we should just list only the TFS supported by linux.

Reply:
I'd really rather avoid putting this in the templates altogether

Reply:
After a discussion with @dansiegel, we're going to move this particular behavior to the SDK, and remove the non-supported frameworks in a target. That will simplify the template significantly.